### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by JB712** [web-ridge/react-native-paper-dates#221] [General] [answered]
I know it is not a funny thing, but I a lot of us appreciate an exhaustive doc with all parameters, there default and possible values

I didn't find it. (may be it is just because I didn't search well, in that case I apologize 💚)

**Comment by iM-GeeKy** [comment, marked as answer]
Hi @JB712, I currently have a pending [PR](https://github.com/web-ridge/react-native-paper-dates/pull/216) that will introduce more dedicated documentation using Docusaurus. Unfortunately, it hasn't been merged, so until @RichardLindhout and I can do so the only documentation is what is in the `README.md`.
